drizzlepac/haputils/photometry_graphics.py

```diff
@@ -86,178 +97,181 @@ def load_data_to_arrays(csv_filename):
+    p1.circle(x='x_index', y=NEW_PHOT_COLUMNS[0], source=sourceDF, fill_alpha=0.5, line_alpha=0.5, size=10, color='green',
               legend_label='Mean Aperture 1 Magnitude Differences')
-    p1.cross(x_index, ap1[2], size=10, color='blue', legend_label='Median Aperture 1 Magnitude Differences')
+    p1.cross(x='x_index', y=NEW_PHOT_COLUMNS[2], source=sourceDF, size=10, color='blue', legend_label='Median Aperture 1 Magnitude Differences')
     info_text = 'Number of datasets: ' + str(num_of_datasets)
     p1.legend.click_policy = 'hide'
 
-    p1.title.text = 'Differences Point - Segment Ap1 Magnitude'
+    p1.title.text = 'Differences (Point - Segment) Aperture 1 Magnitude'
     p1.xaxis.axis_label = 'Index     ' + info_text
-    p1.yaxis.axis_label = 'Difference (Magnitudes)'
+    p1.yaxis.axis_label = 'Difference (ABMag)'
+
+    hover_p1 = HoverTool()
+    hover_p1.tooltips = [("Mean", "@{Ap1 Mean Differences}"), ("StdDev", "@{Ap1 Standard Deviation}"),
+                         ("Median", "@{Ap1 Median Differences}")]
+    p1.add_tools(hover_p1)
 
-    stat_text = ('Mean deltaMAp1_mean: {:6.2f}     Mean deltaMAp1_median: {:6.2f}'.format(means_ap1[0], means_ap1[1]))
+    stat_text = ('<DeltaMagAp1_Mean>: {:6.2f}     <DeltaMagAp1_Median>: {:6.2f}'.format(stat_Ap1[0], stat_Ap1[1]))
     stat_label = Label(x=20, y=20, x_units='screen', y_units='screen', text=stat_text)
     p1.add_layout(stat_label)
 
     p2 = figure()
-    p2.circle(x_index, ap2[0], fill_alpha=0.5, line_alpha=0.5, size=10, color='green',
+    p2.circle(x='x_index', y=NEW_PHOT_COLUMNS[3], source=sourceDF, fill_alpha=0.5, line_alpha=0.5, size=10, color='green',
               legend_label='Mean Aperture 2 Magnitude Differences')
-    p2.cross(x_index, ap2[2], size=10, color='blue', legend_label='Median Aperture 2 Magnitude Differences')
+    p2.cross(x='x_index', y=NEW_PHOT_COLUMNS[5], source=sourceDF, size=10, color='blue', legend_label='Median Aperture 2 Magnitude Differences')
     p2.legend.click_policy = 'hide'
 
-    p2.title.text = 'Differences Point - Segment Ap2 Magnitude'
+    p2.title.text = 'Differences (Point - Segment) Aperture 2 Magnitude'
     p2.xaxis.axis_label = 'Index     ' + info_text
-    p2.yaxis.axis_label = 'Difference (Magnitudes)'
+    p2.yaxis.axis_label = 'Difference (ABMag)'
 
-    stat_text = ('Mean deltaMAp2_mean: {:6.2f}     Mean deltaMAp2_median: {:6.2f}'.format(means_ap2[0], means_ap2[1]))
+    hover_p2 = HoverTool()
+    hover_p2.tooltips = [("Mean", "@{Ap2 Mean Differences}"), ("StdDev", "@{Ap2 Standard Deviation}"),
+                         ("Median", "@{Ap2 Median Differences}")]
+    p2.add_tools(hover_p2)
+
+    stat_text = ('<DeltaMagAp2_Mean>: {:6.2f}     <DeltaMagAp2_Median>: {:6.2f}'.format(stat_Ap2[0], stat_Ap2[1]))
     stat_label = Label(x=20, y=20, x_units='screen', y_units='screen', text=stat_text)
     p2.add_layout(stat_label)
 
     # Display!
     show(row(p1, p2))
```

Comment:
This will automatically bring up the browser to display the generate plot.  Is there a way to simply write out the HTML file without displaying it immediately?  This will be crucial in an automated environment like the pipeline testing or even when we run this code on our own server with 1000 datasets.

Reply:
Fixed in next commit.